source/includes/list-table-shell-properties.rst

```diff
@@ -56,14 +56,23 @@
      - 6
      - The depth to which objects are printed. Setting ``inspectDepth``
        to ``Infinity`` (the javascript object) prints all nested
-       objects to their full depth. 
+       objects to their full depth.
 
    * - ``logLocation``
      - string
      - Depends on your operating system. See :ref:`mdb-shell-view-logs`.
      - Directory where MongoDB Shell writes log files. Specify an
        absolute filepath. See :ref:`mongosh-log-location`.
 
+   * - ``logRetentionDays``
+     - integer or Infinity
```

Comment:
Suggestion:
```
- integer or ``"Infinity"``
```

To clarify that you mean the string "Infinity".

Reply:
It's actually not the string - the command to set to infinity is `config.set("logRetentionDays", Infinity)`